test/e2e/broker_trigger_test.go

```diff
@@ -193,7 +183,7 @@ func TestDefaultBrokerWithManyTriggers(t *testing.T) {
 		}
 		// Create sender pod.
 		senderPodName := name("sender", eventToSend.Type, eventToSend.Source)
```

Comment:
`name()` will need to change to include the test's name or some randomness, as multiple tests may create a sender that sends the same event type and source.

Reply:
No longer a problem after we make each test case running in separate namespace.